### PR TITLE
Add set-repository-policy to pull with secondary-account-ids

### LINKED ISSUE #28: Whitelist AWS account for image pull
### Orb version

Applies to >=4.0.2

### What happened

Would be neat to have parameters for AWS account IDs to give secondary access to created repos. If a config has `create-repo: true`, there could be another property `secondary-account-ids` that takes a list of AWS account IDs to add to a policy that allows for image pulling from those accounts.

### Expected behavior

By using the orb and listing `create-repo: true` and `secondary-account-ids`, my build should build and push an image to a newly-created repo, and my secondary accounts should be able to pull from the newly-created repo.

### PR DESCRIPTION
**Pull request by furlanf**
### Checklist

- [X] All new jobs, commands, executors, parameters have descriptions
- [ ] Examples have been added for any significant new features
- [ ] README has been updated, if necessary

### Motivation, issues
Resolves #28

### Description
- add a new command to set a repository policy to be able to pull with secondary id's

### CODE REVIEW

File: src/commands/set-policy.yml [hunk 1]
```diff
@@ -0,0 +1,68 @@
+description: "Set policy to pull a repository with secondary ids"
+parameters:
+
+  secondary-account-ids:
+    description: A comma-separated string containing secondary-account-ids
```
Comment (KyleTryon) [inline src/commands/set-policy.yml]: The "env_var_name" type should not allow commas. This may have to be left as a string if comma-separated is required.

Comment (KyleTryon) [reply src/commands/set-policy.yml]: Ahh and I just realized this means the env var value is comma-separated. My mistake.

Comment (furlanf) [reply src/commands/set-policy.yml]: Yes, it is an env_var with value comma-separated.
I didn't know if I use an env_var or a string.

**Comment by furlanf** [comment]
I'll create a use-case and maybe change the name to `set-pull-policy`, what do you think?

**Pull request closed without merging**